pages/about.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import Image from "next/image";
+import aboutImg from "@/public/assets/about/about-image.png";
+import tabData from "../data/about/tab";
```

Comment:
I think we can leave every information on about page in the about.js file instead of creating a folder for it